### PR TITLE
sendMail lib installed and added to donate route

### PR DESCRIPTION
**Pull request by TheoBoschetti**

### CODE REVIEW

**Review by val-fcn — DISMISSED** [review]

File: backend/routes/donation.js [hunk 1]
```diff
@@ -88,16 +89,21 @@ router.post("/", (req, res) => {
...
                           "Error has occured during the post of the new donation !"
                         );
-                    return res
-                      .status(201)
-                      .send("Donation posted successfully !");
+                    res.status(201).send("Donation posted successfully !");
+                    sendMail({
+                      from: "Team.Meex@meex.com",
+                      to: email,
+                      replyTo: "Team.Meex@meex.com",
```
Comment (LoicBrassart) [inline backend/routes/donation.js]: (By convention, you should use 'no-reply@meex.com' if the address doesn't exist)

File: backend/routes/donation.js [hunk 2]
```diff
@@ -88,16 +89,21 @@ router.post("/", (req, res) => {
...
+                      subject: "Thank you for your donation !",
+                      html: `Thank you! You just donated ${donation_value}€ and by doing this you participated to have a better world !`
```
Comment (LoicBrassart) [inline backend/routes/donation.js]: (While correct, this message looks strange in the case of a 0€ donation, doesn't it ? To make it better, you could create another message and switch it in this specific case)

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**